bin/docserv

```diff
@@ -698,5 +706,8 @@ ch.setFormatter(formatter)
 logger.addHandler(ch)
 
 if __name__ == "__main__":
-    docserv = Docserv(sys.argv)
-    docserv.start()
+    if "--help" in sys.argv:
+        print_help()
+    else:
+        docserv = Docserv(sys.argv)
+        docserv.start()
```

Comment:
Should we return a return code?

Reply:
The script already implicitly returned the correct code (0). Now I added a line to explicitly return 0.